app/imports/ui/pages/EditEvent.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import swal from 'sweetalert';
+import { Card, Col, Container, Row } from 'react-bootstrap';
+import { AutoForm, ErrorsField, HiddenField, LongTextField, SelectField, SubmitField, TextField } from 'uniforms-bootstrap5';
+import { Meteor } from 'meteor/meteor';
+import { useTracker } from 'meteor/react-meteor-data';
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import { useParams } from 'react-router';
+import LoadingSpinner from '../components/LoadingSpinner';
+import { Events } from '../../api/event/Event';
+
+const bridge = new SimpleSchema2Bridge(Events.schema);
+
+/* Renders the EditEvent page for editing a single document. */
+const EditEvent = () => {
+  // Get the documentID from the URL field. See imports/ui/layouts/App.jsx for the route containing :_id.
+  const { _id } = useParams();
+  // console.log('EditEvent', _id);
+  // useTracker connects Meteor data to React components. https://guide.meteor.com/react.html#using-withTracker
+  const { doc, ready } = useTracker(() => {
+    // Get access to Contact documents.
+    const subscription = Meteor.subscribe(Events.adminPublicationName);
+    const subscription2 = Meteor.subscribe(Events.companyPublicationName);
```

Comment:
I'd recommend using different variable names here

Reply:
What do you mean 'different variable names here'?